### DISCUSSION TITLE
How to exit fullscreen with --display=spice?

### DISCUSSION THREAD

**Discussion by wusspuss** [quickemu-project/quickemu#970] [Quickemu] [answered]
Ctrl-Alt-G has no effect. No applet on top like with virt-manager either. How do I leave fullscreen?

**Comment by wusspuss** [comment, marked as answer]
Ok it turned out rather tricky: ctrl+alt, then shift+f12. I think it would be nice if that was at least printed prior to launch.

**Comment by moisleyi** [comment]
Works for me with X11...

SHIFT_L+F12 # exit fullscreen
SHIFT+F11 # fullscreen
